packages/slice-machine/components/Forms/types.ts

```diff
@@ -0,0 +1,3 @@
+export interface SliceModalValues {
```

Comment:
Hum, I wanted to discuss that, I like your idea. Maybe even more, we could just use the folder name as the prefix. Here it's Forms with an s (not the best I guess for our futur clean Form), so it would be "formsTypes.ts" and "formsValidator.ts", what do you think?